renku/core/models/datasets.py

```diff
@@ -395,6 +395,13 @@ def editable(self):
         data = {field_: obj.pop(field_) for field_ in self.EDITABLE_FIELDS}
         return data
 
+    def contains(self, files):
```

Comment:
Why is it miss leading? It follows a common functionality pattern which can be found in standard library (and in other languages standard library).

Reply:
Since it accepts multiple files I expect `contains` return true only if all files are contained. This function return true when any of files is in the dataset.

Reply:
Ok, renamed to `contains_any`.